sbom.cdx.json

```diff
@@ -0,0 +1,39 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "components": [
+    {
+      "type": "library",
+      "bom-ref": "pkg:github/provider-corner/libprov@@VCS_TAG@",
+      "cpe": "cpe:2.3:a:provider-corner:libprov:@VCS_TAG@:*:*:*:*:*:*:*",
+      "name": "libprov",
+      "version": "@VCS_VERSION@",
+      "description": "Small library of helpers for OpenSSL 3 providers",
+      "authors": [
+        {
+          "name": "@VCS_SBOM_AUTHORS@"
+        }
+      ],
+      "supplier": {
+        "name": "Richard Levitte",
+        "url": [
+          "https://github.com/levitte"
+        ]
+      },
```

Comment:
Please remove this.  It's not mandatory, and it's not necessary.

Reply:
Done.